cli/tools/bundle.rs

```diff
@@ -67,13 +69,18 @@ pub async fn bundle(
 
   let operation = |(ps, graph): (ProcState, Arc<deno_graph::ModuleGraph>)| {
     let out_file = bundle_flags.out_file.clone();
+    let source_file = bundle_flags.source_file.clone();
     async move {
       // at the moment, we don't support npm specifiers in deno bundle, so show an error
       error_for_any_npm_specifier(&graph)?;
 
-      let bundle_output = bundle_module_graph(graph.as_ref(), &ps)?;
+      let mut bundle_output = bundle_module_graph(graph.as_ref(), &ps)?;
       log::debug!(">>>>> bundle END");
 
+      if let Some(shebang) = shebang_file(source_file.clone()) {
```

Comment:
Instead of reading from the filesystem, we could look at `graph`, which already contains the file text in memory. It's available at `graph.get(&graph.roots[0].0).unwrap().maybe_source` (I believe there is only ever one root when using bundle, so unwrapping there is probably ok).

Reply:
thx